Packer 1.4.3 crash when building windows instances

When building windows images with packer v1.4.3, we are encountering an error at the `Creating windows user for instance...` stage. 

- Packer version: v1.4.3 : Has been tested with v1.4.2 and v1.4.1 and the error did not occur.
  The packer instance that we are using also works fine when building other OS images (tested with rhel/CentOS).  
- [Crash.log gist](https://gist.github.com/P15223971/156599e8d9d02091749a888dce27f063)
- The main issue appears to be as follows:

```==> Builds finished but no artifacts were created.
panic: runtime error: invalid memory address or nil pointer dereference
2019/09/03 15:05:33 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1cc79f7]
2019/09/03 15:05:33 packer: 
2019/09/03 15:05:33 packer: goroutine 211 [running]:
2019/09/03 15:05:33 packer: github.com/hashicorp/packer/builder/googlecompute.(*StepCreateWindowsPassword).Run(0xc0002ce1e0, 0x3fadba0, 0xc000372f40, 0x3f936a0, 0xc00079c480, 0x0)
2019/09/03 15:05:33 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/step_create_windows_password.go:63 +0x657
2019/09/03 15:05:33 packer: github.com/hashicorp/packer/helper/multistep.(*BasicRunner).Run(0xc00079c510, 0x3fadba0, 0xc000372f40, 0x3f936a0, 0xc00079c480)
2019/09/03 15:05:33 packer: 	/Users/mmarsh/Projects/packer/helper/multistep/basic_runner.go:67 +0x1ec
2019/09/03 15:05:33 packer: github.com/hashicorp/packer/builder/googlecompute.(*Builder).Run(0xc00038e0a0, 0x3fadba0, 0xc000372f40, 0x3fc97a0, 0xc0002ce0a0, 0x3f46f40, 0xc00063e020, 0x40b9b9, 0xc000479890, 0x30, ...)
2019/09/03 15:05:33 packer: 	/Users/mmarsh/Projects/packer/builder/googlecompute/builder.go:86 +0x95a
2019/09/03 15:05:33 packer: github.com/hashicorp/packer/packer/rpc.(*BuilderServer).Run(0xc0001de2d0, 0x1, 0xc0007749b0, 0x0, 0x0)
2019/09/03 15:05:33 packer: 	/Users/mmarsh/Projects/packer/packer/rpc/builder.go:109 +0x231
2019/09/03 15:05:33 packer: reflect.Value.call(0xc0003ad380, 0xc0003c43e8, 0x13, 0x3998dd5, 0x4, 0xc0000d5f18, 0x3, 0x3, 0xc000388000, 0x0, ...)
2019/09/03 15:05:33 packer: 	/usr/local/go/src/reflect/value.go:447 +0x461
2019/09/03 15:05:33 packer: reflect.Value.Call(0xc0003ad380, 0xc0003c43e8, 0x13, 0xc00024a718, 0x3, 0x3, 0x0, 0xc00082cf60, 0x20)
2019/09/03 15:05:33 packer: 	/usr/local/go/src/reflect/value.go:308 +0xa4
2019/09/03 15:05:33 packer: net/rpc.(*service).call(0xc0003b3180, 0xc0000ca370, 0xc000492010, 0xc000492020, 0xc0003bc580, 0xc00079a0e0, 0x2ecd180, 0xc0007749ac, 0x18a, 0x2e1fce0, ...)
2019/09/03 15:05:33 packer: 	/usr/local/go/src/net/rpc/server.go:384 +0x14e
2019/09/03 15:05:33 packer: created by net/rpc.(*Server).ServeCodec
2019/09/03 15:05:33 packer: 	/usr/local/go/src/net/rpc/server.go:481 +0x42b
2019/09/03 15:05:33 [INFO] (telemetry) ending googlecompute
2019/09/03 15:05:33 ui error: 2019-09-03T15:05:33Z: Build 'googlecompute' errored: unexpected EOF
2019/09/03 15:05:33 machine readable: error-count []string{"1"}
2019/09/03 15:05:33 ui error: 
==> Some builds didn't complete successfully and had errors:
2019/09/03 15:05:33 machine readable: googlecompute,error []string{"unexpected EOF"}
2019/09/03 15:05:33 ui error: --> googlecompute: unexpected EOF
2019/09/03 15:05:33 ui: 
==> Builds finished but no artifacts were created.
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 [INFO] (telemetry) Finalizing.
2019/09/03 15:05:33 waiting for all plugin processes to complete...
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited
2019/09/03 15:05:33 /bin/packer: plugin process exited



!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.

[1]: https://github.com/hashicorp/packer/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
 
Any assistance would be appreciated, and please let me know if there is any further details that would help.

Thanks.

--- a/builder/googlecompute/step_create_windows_password.go
+++ b/builder/googlecompute/step_create_windows_password.go
@@ -55,12 +55,17 @@ func (s *StepCreateWindowsPassword) Run(ctx context.Context, state multistep.Sta
 	buf := make([]byte, 4)
 	binary.BigEndian.PutUint32(buf, uint32(priv.E))
 
+	email := ""
+	if c.Account != nil {
+		email = c.Account.Email
+	}
+
 	data := WindowsPasswordConfig{
 		key:      priv,
 		UserName: c.Comm.WinRMUser,
 		Modulus:  base64.StdEncoding.EncodeToString(priv.N.Bytes()),
 		Exponent: base64.StdEncoding.EncodeToString(buf[1:]),
-		Email:    c.Account.Email,
+		Email:    email,
 		ExpireOn: time.Now().Add(time.Minute * 5),
 	}
 

